calculator/UnknownOperatorException.cpp

```diff
@@ -0,0 +1,5 @@
+#include "UnknownOperatorException.h"
+
+const char* UnknownOperatorException::what() const noexcept {
+	return "Unknown Operator";
```

Comment:
What unknown operator?
Maybe receive it in the exception's constructor and present it to the user in the return value of what()? :)